docs/conda_windows_install.md

```diff
@@ -17,3 +17,67 @@ conda deactivate
 ```
```

Comment:
Add new lines for installing ipykernel and adding the env to Jupyter to the code chunk at line 16.